packages/commands-http/src/request.ts

```diff
@@ -0,0 +1,85 @@
+import { Command } from '@seagull/commands'
+import fetch from 'node-fetch'
+import { Http, RequestConfig } from './index'
+import SeedStorage from './seedStorage'
+
+/**
+ * Http request command.
+ */
+export class Request<T> extends Command<T> {
+  executeConnected = this.executeCloud
+  executeEdge = this.executeCloud
+
+  private seed: SeedStorage<T>
+
+  /**
+   * Creates an http request command.
+   * @param config Http request command configuration.
+   */
+  constructor(public config: RequestConfig) {
+    super()
+    if (!config) {
+      throw new Error('HttpCommand: You have to specify a config!')
+    }
+    if (!this.config.url) {
+      throw new Error('HttpCommand: The url in the config is not specified!')
+    }
+    this.config = config
+    this.seed = SeedStorage.createByRequest(config.url, config.init)
+  }
+
+  async executeCloud(): Promise<T> {
+    const response = await fetch(this.config.url, this.config.init)
+    switch (this.config.parseBody) {
```

Comment:
For our usecase we need support for the raw data. What do you think of exposing `response.arrayBuffer()`, 
`response.blob()` and `response.buffer()`? Or even `response.body`? See https://github.com/bitinn/node-fetch#interface-body for a complete list of return types.

Reply:
Sure, wip =)

Reply:
exposing response.body is only possible when returning a different response, so it could for example be a parse mode. Not sure if this is a good idea in terms of encapsulation.

Reply:
How about returning the base64 string as used within visir-commands. This would allow the test to use the base64 string as fixture.
Point is, as soon as i expose the whole body object, i'll have to implement the seed generation for all cases it implements ...

Reply:
implemented like described